robot/EDA/resources/EDA.robot

```diff
@@ -175,6 +212,61 @@ Create Contact
     Store Session Record      Contact  ${contact_id}
     [return]                  ${contact_id}
     
+
+
+
+API Create Advising Or Walkin Appointment
```

Comment:
@dashattuck @boakley Should this really be in the EDA robot tests, since it's specific to SAL?

Reply:
Nope - no need, but this is a candidate already for a complete rewrite - so it will get cleaned up on a different ticket.

Reply:
Why not clean it up now? What's the point of adding a SAL test in an EDA test suite?

Reply:
I removed this code.

Reply:
I removed this function